dev/com.ibm.ws.persistence/src/com/ibm/wsspi/persistence/internal/DatabaseStoreImpl.java

```diff
@@ -152,7 +155,7 @@ private SpecialEntitySet recognizeSpecialEntityPackage(String className) {
 
     @Activate
     @Trivial
-    protected void activate(ComponentContext context) throws Exception {
+    public DatabaseStoreImpl(ComponentContext context) throws Exception {
```

Comment:
Method name is changed, but the trace entry/exit still shows its name as "activate"

Reply:
Good catch.  I removed `@Trivial` and the hard coded trace statements that were printing entry/exit themselves with the properties content.  Instead the constructor is passed the `Map<String,?>` component properties directly so we can use the standard entry/exit tracing as-is.  Output will look like this now for trace:

```
[12/14/23, 10:42:05:152 CST] 00000048 id=00000000 com.ibm.wsspi.persistence.internal.DatabaseStoreImpl         > <init> Entry  
                                                                                                               {osgi.ds.satisfying.condition.target=(osgi.condition.id=true), tablePrefix=DATA, component.name=com.ibm.ws.persistence.databaseStore, id=application[DataTestApp]/module[DataTestApp.war]/databaseStore[java:module/jdbc/env/DerbyDataSourceRef], DataSourceFactory.target=(id=application[DataTestApp]/module[DataTestApp.war]/databaseStore[java:module/jdbc/env/DerbyDataSourceRef]/ResourceFactory), AuthData.cardinality.minimum=0, createTables=true, dropTables=true, service.factoryPid=com.ibm.ws.persistence.databaseStore, AuthData.target=(service.pid=${authDataRef}), service.pid=com.ibm.ws.persistence.databaseStore_1, keyGenerationStrategy=AUTO, NonJTADataSourceFactory.cardinality.minimum=0, component.id=336, config.displayId=application[DataTestApp]/module[DataTestApp.war]/databaseStore[java:module/jdbc/env/DerbyDataSourceRef], NonJTADataSourceFactory.target=(&(service.pid=${nonTransactionalDataSourceRef})(transactional=false))}
[12/14/23, 10:42:05:152 CST] 00000048 id=84c3fa4c com.ibm.wsspi.persistence.internal.DatabaseStoreImpl         < <init> Exit 
```